src/cascadia/TerminalControl/UiaTextRange.cpp

```diff
@@ -34,6 +34,7 @@ std::deque<UiaTextRange*> UiaTextRange::GetSelectionRanges(_In_ IUiaData* pData,
             {
                 UiaTextRangeBase* temp = ranges[0];
                 ranges.pop_front();
+                temp->Release();

```

Comment:
Where'd this come from?

Reply:
this was a code correctness issue that the new “unused locals” warning caught. Ranges is an unprotected vector of raw pointer types, so ref counting wasn’t being done on it.